base/src/main/java/com/mindera/skeletoid/logs/LoggerManager.kt

```diff
@@ -128,7 +131,7 @@ internal class LoggerManager : ILoggerManager {
             val log = String.format(
                 LOG_FORMAT_4ARGS,
                 tag,
-                LogAppenderUtils.getObjectHash(tag),
+                LogAppenderUtils.getObjectHash(invokingClass),
```

Comment:
We could do better. Since `getObjectHash` is returning empty string this can end up having two empty spaces in between tag and thread name.

We could do the following

```
if (logAppenders.isEmpty()) return

val headers = listOfNotNull(
    tag,
    LogAppenderUtils.getObjectHash(invokingClass),
    ThreadUtils.currentThreadName,
).joinToString(separator = " ")

val log = String.format(LOG_FORMAT, headers, LogAppenderUtils.getLogString(*text))

pushLogToAppenders(priority, t, log)
```

And having `getObjectHash` return a nullable string 

```
@JvmStatic
fun getObjectHash(obj: Any?): String? =
    obj?.let {
        val stringBuilder = StringBuilder()
        stringBuilder.append("[")
        stringBuilder.append(obj.javaClass.simpleName)
        stringBuilder.append("#")
        stringBuilder.append(obj.hashCode())
        stringBuilder.append("] ")
        stringBuilder.toString()
    }
```